group_vars/ncn/services.suse.yml

```diff
@@ -24,23 +24,49 @@
 ---
 services:
   - enabled: true
+    name: cfs-state-reporter.service
+    state: stopped
+  - enabled: false
     name: cloud-config.service
     state: stopped
-  - enabled: true
+  - enabled: false
     name: cloud-init.service
     state: stopped
-  - enabled: true
+  - enabled: false
     name: cloud-init-local.service
     state: stopped
-  - enabled: true
+  - enabled: false
+    name: cloud-init-oneshot.service
+    state: stopped
+  - enabled: false
     name: cloud-final.service
     state: stopped
   - enabled: true
-    name: csm-node-identity.service
+    name: getty@tty1.service
+    state: started
+  - enabled: true
+    name: goss-servers
+    state: stopped
+  - enabled: true
+    name: issue-generator.service
+    state: stopped
+  - enabled: true
+    name: metalfs.service
     state: stopped
   - enabled: true
-    name: kubelet.service
+    name: rasdaemon.service
+    state: stopped
+  - enabled: true
+    name: rollback.service
     state: stopped
   - enabled: false
     name: spire-agent.service
     state: stopped
+# FIXME: Either handle this map, or undo this.
```

Comment:
Make a ticket for this?

Reply:
This is actually something that should be yanked, it's handled now.